vllm/model_executor/layers/fused_moe/fused_moe_gptq.py

```diff
@@ -0,0 +1,146 @@
+"""Fused MoE utilities for GPTQ."""
+import functools
+import torch
+
+from typing import Any, Dict, Optional
+from vllm import _custom_ops as ops
+
+from .fused_moe import fused_topk, moe_align_block_size, try_get_optimal_moe_config
+
+
+def fused_moe_gptq(
```

Comment:
can you call this `fused_moe_marlin`? We want to separate the naming of the kernel from the algorithm